SU2_CFD/include/SU2_CFD.hpp

```diff
@@ -39,7 +39,7 @@
 #pragma once
 
 #include "../../Common/include/mpi_structure.hpp"
-#include <CLI11.hpp>
+#include "../../externals/CLI11/CLI11.hpp"
```

Comment:
Same comment as in another PR, this should probably be `#include "CLI11.hpp"` and the path added to the include directories of your IDE

Reply:
Yep, I would also suggest doing it like that.

Reply:
Will revert. I also reverted this in the CGNS PR. Yep, Xcode is picky, but we can just add an extra search path inside the project file